subjects/graphql/lib/schema.json

```diff
@@ -0,0 +1,136 @@
+{
```

Comment:
why not query the schema from graphQL instead of having it hardcoded ?
This is a good way to make sure it's out of date soon